fandogh_cli/base_commands.py

```diff
@@ -19,16 +20,19 @@ def invoke(self, ctx):
             self._check_for_new_version()
             return super(FandoghCommand, self).invoke(ctx)
         except (FandoghAPIError, AuthenticationError) as exp:
+            collect(self, ctx, exp)
             debug('APIError. status code: {}, content: {}'.format(
                 exp.response.status_code,
                 exp.response.content))
             click.echo(format_text(exp.message, TextStyle.FAIL), err=True)
             exit(1)
         except VersionException as exp:
+            collect(self, ctx, exp)
             click.echo(format_text("New Version of {} is available, please update to continue "
                                    "using Fandogh services using : `pip install {} --upgrade`".format(NAME, NAME),
                                    TextStyle.FAIL))
         except Exception as exp:
+            collect(self, ctx, exp)
```

Comment:
Everyone(other softwares) ask for permission before collect anonymous statistics and stuff, shouldn't we?

Reply:
I was going to do it. but we are just reporting the errors, not tracking the user behavior, I believe it's fine to not ask for permission for such info.